db/osqlsession.c

```diff
@@ -156,10 +153,8 @@ static void _destroy_session(osql_sess_t **prq, int phase)
     case 2:
         Pthread_mutex_destroy(&rq->mtx);
     case 3:
-        Pthread_mutex_destroy(&rq->clients_mtx);
-    case 4:
         Pthread_mutex_destroy(&rq->completed_lock);
-    case 5:
+    case 4:
         free(rq);
```

Comment:
Would there be any value to atomically setting the rq->clients value to something obviously invalid here (e.g. -1, etc)?

Reply:
We free rq in next line. I don't think setting client to -1 makes any difference.